### ISSUE TITLE
Update FAQ with 21 questions? 

### ISSUE THREAD

**Issue by dergigi** [nostr-resources/nostr-resources.github.io#66]
Some good stuff in there: https://njump.me/nevent1qqsvhj5xgpn3gg0zxyx905l36s23myk95x723m2ast0vv90cjdxf7vspzpmhxue69uhkummnw3ezumt0d5hsygx3v2jncwctldwra0tc04as3l4typ43zgmzaj394g53y5wdwrlzy5psgqqqqqqsp4psqq

**Comment by dergigi** [comment]
Hm, not sure. What do you think @Dawn11111111 ?

**Comment by Dawn11111111** [comment]
I don't know, I think 21 FAQs is an awful lot for top placement on the page. The questions are good though. Maybe we need a rearrange?

**Comment by dergigi** [comment]
Closing this — most of the 21 questions are already covered throughout the page (just not all in the FAQ section specifically). The current FAQ works well for onboarding as-is.

**Comment by dergigi** [comment]
> Maybe we need a rearrange?

Yeah I don't hate the idea of revamping the site in general and making it more structured. That's a bigger task though.

**Comment by Dawn11111111** [comment]
Good call. I'll continue to keep on eye on things. There's a couple links in question now but they're veterans that are freshly expired so I figure they need some time. If you ever do decide its time for a major overhaul, I would be thrilled to help with that.

**Comment by dergigi** [comment]
I've tracked an issue for a potential v2:

- https://github.com/nostr-resources/nostr-resources.github.io/issues/112
